moveit_ros/planning_interface/planning_scene_interface/include/moveit/planning_scene_interface/planning_scene_interface.h

```diff
@@ -142,6 +142,11 @@ class PlanningSceneInterface
       consider using `applyCollisionObjects` instead. */
   void removeCollisionObjects(const std::vector<std::string>& object_ids) const;
 
+  /** \brief Remove all the collision and attached objects from the world via the planning scene of the move_group node synchronously.
+
+      Other PlanningSceneMonitors will NOT receive the update unless they subscribe to move_group's monitored scene */
+  bool clearScene();
```

Comment:
There is nothing else to clear than the scene.
```suggestion
  bool clear();
```